schemas/mozilla_nimbus_schemas/experiments/experiments.py

```diff
@@ -0,0 +1,70 @@
+from typing import List, Optional, Union
+
+from pydantic import BaseModel
+
+
+class BucketConfig(BaseModel):
+    randomizationUnit: str
```

Comment:
Guessing the camelCase is because these schemas were originally defined in typescript, but just commenting in case you want to change.

Reply:
No no yeah we have to keep it in camel case 🐫 🐍